networking/hub-region.v1.json

```
@@ -455,6 +457,357 @@
+                                                "port": 443
+                                            }
+                                        ],
+                                        "targetFqdns": [
+                                            "releases.hashicorp.com"
+                                        ],
+                                        "webCategories": [],
+                                        "destinationAddresses": [],
+                                        "fqdnTags": []
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                },
+                {
+                    "type": "Microsoft.Network/firewallPolicies/ruleCollectionGroups",
+                    "apiVersion": "2020-11-01",
+                    "name": "[concat(variables('fwPoliciesName'), '/DefaultNetworkRuleCollectionGroup')]",
+                    "location": "[parameters('location')]",
+                    "dependsOn": [
+                        "[variables('fwPoliciesName')]",
+                        "[resourceId('Microsoft.Network/firewallPolicies/ruleCollectionGroups', variables('fwPoliciesName'), 'DefaultApplicationRuleCollectionGroup')]"
+                    ],
+                    "properties": {
+                        "priority": 200,
+                        "ruleCollections": [
+                            {
+                                "name": "allow-ntp",
+                                "ruleCollectionType": "FirewallPolicyFilterRuleCollection",
+                                "action": {
+                                    "type": "Allow"
+                                },
+                                "priority": 100,
+                                "rules": [
+                                    {
+                                        "name": "ntp",
+                                        "ruleType": "NetworkRule",
+                                        "ipProtocols": [
+                                            "UDP"
+                                        ],
+                                        "sourceAddresses": [],
+                                        "sourceIpGroups": [
+                                            "[resourceId('Microsoft.Network/ipGroups', variables('imageBuilderIpGroupName'))]"
+                                        ],
+                                        "destinationAddresses": [
+                                            "*"
+                                        ],
+                                        "destinationPorts": [
+                                            "123"
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                }
+          ]
```

```suggestion
            ]
```
Indent is off, making an unremarkable difference between the files, which we want to strongly avoid.

---

As mentioned above, this change needs to be in v2 as well.